File: Code Chef/JavaScript/300-399/324 Tasty Decisions TASTEDEC.js
process.stdin.setEncoding("utf-8");

const strToInt = (strArray) => {
    strArray.shift();
    const intArray = strArray.map((strItem) => strItem.split(" ").map(Number));
    
    return intArray;
};

const whatIsTasty = (chocolatesInPack, candiesInPack) => {
    const tasteRatingOfChocolate = chocolatesInPack * 2;
    const tasteRatingOfCandy = candiesInPack * 5;
    
    (tasteRatingOfChocolate === tasteRatingOfCandy)
        ? console.log("Either")
        : (tasteRatingOfChocolate > tasteRatingOfCandy)
            ? console.log("Chocolate")
            : console.log("Candy");
};

process.stdin.on("data", (input) => {
    const testcases = strToInt(input.trim().split("\n"));
    
    testcases.forEach((testcase) => {
        const [chocolatesInPack, candiesInPack] = testcase;
        whatIsTasty(chocolatesInPack, candiesInPack);
    });
});

File: Code Chef/C Programming/300-399/324 Tasty Decisions TASTEDEC.c
#include <stdio.h>
int main(void)
{
	int t,x,y;
	scanf("%d",&t);
	for(int i=0;i<t;i++)
	{
	    scanf("%d%d",&x,&y);
	    if((2*x)==(5*y))
	    {
	        printf("Either\n");
	    }
	    else if((2*x)>(5*y))
	    {
	        printf("Chocolate\n");
	    }
	    else
	    {
	        printf("Candy\n");
	    }
	}
	return 0;
}